script.js

```diff
@@ -4,54 +4,53 @@ const form = document.querySelector('form');
 const title = document.querySelector('.title');
 const author = document.querySelector('.author');
 
-const information = JSON.parse(localStorage.getItem('books')) || [];
+class BookList {
+  information = JSON.parse(localStorage.getItem('books')) || [];
 
+  storeBook() {
+    const newBook = {
+      name: title.value,
+      author: author.value,
+    };
+    this.information.push(newBook);
+    localStorage.setItem('books', JSON.stringify(this.information));
+    return newBook;
+  }
 
-const displayData = function (){
-    information.forEach((book, index) => {
-        const bookTitle = document.createElement('p');
-        const bookAuthor = document.createElement('p');
-        const remove = document.createElement('button');
-        remove.textContent = "Remove";
-        remove.type = 'button';
-        remove.classList.add('remove');
-        const bookDiv = document.createElement('div');
-        const horizontalBar = document.createElement('hr');
-        bookTitle.innerHTML = book.name;
-        bookAuthor.innerHTML = book.Author;
-        bookDiv.append(
-            bookTitle,
-            bookAuthor,
-            remove,
-            horizontalBar
-        );
-        bookDisplayContainer.appendChild(bookDiv);
-        remove.addEventListener('click', () => {
-            bookDisplayContainer.removeChild(remove.parentElement);
-            information.splice(index, 1);
-            localStorage.setItem('books', JSON.stringify(information));
-        })
+  displayBooks() {
+    this.information.forEach((book, index) => {
+      const bookTitle = document.createElement('p');
+      const remove = document.createElement('button');
+      remove.textContent = 'Remove';
+      remove.type = 'button';
+      remove.classList.add('remove');
+      const bookDiv = document.createElement('div');
+      bookDiv.classList.add('book-div');
+      bookTitle.innerHTML = `${book.name} by ${book.author}`;
+      bookDiv.append(bookTitle, remove);
+      bookDisplayContainer.appendChild(bookDiv);
+      remove.addEventListener('click', () => {
+        bookDisplayContainer.removeChild(remove.parentElement);
+        this.information.splice(index, 1);
+        localStorage.setItem('books', JSON.stringify(this.information));
+        location.reload();
+      });
```

Comment:
- [ ] It would be nice if you could extract this remove functionality into a removeBook function that is created inside the class and then when you add an event listener to the remove button, you invoke the removeBook function. Your implementation is working but the idea is for you to have separate methods created in the class and used whenever you want to and it's required you have methods to add and remove. Kindly work on that 👍 
- [ ] Doing that will see that your class has storeBook, removeBook, and displayBooks methods. That is the deal of using classes to have a separation of concerns 👍